src/object.d

```diff
@@ -269,7 +269,8 @@ void setSameMutex(shared Object ownee, shared Object owner)
  */
 struct Interface
 {
-    TypeInfo_Class   classinfo;  /// .classinfo for this interface (not for containing class)
+    /// Class info returned by `typeid` for this interface (not for containing class)
+    TypeInfo_Class   classinfo;
```

Comment:
There is a difference for an `interface I`:
- I.classinfo -> TypeInfo_Class
- typeid(I) -> TypeInfo_Interface

But using classinfo and typeid on an instance of I both yield TypeInfo_Class.

Reply:
Reminds me of https://github.com/dlang/dmd/pull/4711

Reply:
Yes there are a few issues mentioned here: https://github.com/dlang/dmd/pull/11033#issuecomment-613187083